recipes/libtensorlight/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "3.0.1" %}
+{% set sha256 = "1ec596bbd160b4df413bccb42e88d876e098aa62899f6b1bb50003d5552bf9cb" %}
+{% set build = 0 %}
+
+package:
+  name: libtensorlight
+  version: {{ version }}
+
+source:
+  url: https://github.com/adc-connect/libtensor/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - mkl-devel {{ mkl }}
+    - llvm-openmp            # [osx]
+    - libgomp                # [linux]
+    - cmake >=3.2
+    - ninja
+
+  host:
+    - mkl-devel
+
+  run:
+    - libgomp                # [linux]
+
+test:
+  requires:
+      - mkl 
+  commands:
+      - test -f $PREFIX/lib/libtensorlight$SHLIB_EXT
+      - ldd -v $PREFIX/lib/libtensorlight$SHLIB_EXT                 # [linux]
+      - otool -L $PREFIX/lib/libtensorlight$SHLIB_EXT               # [osx]
+      - conda inspect linkages --show-files --groupby=dependency libtensorlight
+      - conda inspect objects -p $PREFIX $PKG_NAME                  # [osx]
+
+outputs:
+  - name: libtensorlight
```

Comment:
If this is a single output, please don't use outputs

Reply:
When using `files` directly, I get "The top level meta key files is unexpected", that's why I used `outputs`.